src/System.Devices.Gpio/System.Devices.Gpio.Samples/Program.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Diagnostics;
```

Comment:
nit: consider renaming the folder to just Samples rather than System.Devices.Gpio.Samples.

So, it would be `src/System.Devices.Gpio/Samples/Program.cs`